client/components/catalog/TagFilter.vue

```diff
@@ -6,22 +6,25 @@
       </v-btn>
     </template>
     <v-list>
-      <v-list-item
-        v-for="it in options"
-        :key="it.id"
-        @click="$emit('update', it)">
-        <v-list-item-action class="mr-2">
-          <v-checkbox :value="it.isSelected" />
-        </v-list-item-action>
-        <v-list-item-content class="text-left">
-          <v-list-item-title>{{ it.name }}</v-list-item-title>
-        </v-list-item-content>
-      </v-list-item>
-      <v-list-item v-if="!options.length">
-        <v-list-item-content class="text-left">
-          <v-list-item-title>No available tags</v-list-item-title>
-        </v-list-item-content>
-      </v-list-item>
+      <v-text-field
```

Comment:
v-text field probably shouldn't be part of list 🤔 and list items should be direct children.